gomint-server/src/main/java/io/gomint/server/world/generator/vanilla/client/Client.java

```diff
@@ -678,6 +623,7 @@ private void move(Location target) {
         movePlayer.setYaw(target.getYaw());
         movePlayer.setPitch(target.getPitch());
         movePlayer.setMode(PacketMovePlayer.MovePlayerMode.TELEPORT);
+        movePlayer.setTick(0);
```

Comment:
Why a static value of 0 here? We could use the server tick here

Reply:
You cant send ticks as a client

Reply:
Only when you use the PlayerAuthInputPacket, but then we would have to rewrite how the bot is moving